esphome/components/sim800l/__init__.py

```diff
@@ -1,8 +1,8 @@
 import esphome.codegen as cg
 import esphome.config_validation as cv
 from esphome import automation
-from esphome.const import CONF_ID, CONF_TRIGGER_ID
-from esphome.components import uart
+from esphome.const import CONF_ID, CONF_TRIGGER_ID, UNIT_DECIBEL_MILLIWATT, DEVICE_CLASS_SIGNAL_STRENGTH, STATE_CLASS_MEASUREMENT
+from esphome.components import sensor, uart
 
 DEPENDENCIES = ["uart"]
```

Comment:
This is required otherwise the sensor cpp code will not be added to the firmware.
```suggestion
AUTO_LOAD = ["sensor"]
DEPENDENCIES = ["uart"]
```